src/main/docker/docker-compose.yml

```diff
@@ -101,7 +101,7 @@ services:
        - rabbitmq.bindingKey=#
        - rabbitmq.autoDelete=false
        - rabbitmq.createExchangeIfNotExisting=true
-       - rabbitmq.consumerName=messageConsumer
+       - rabbitmq.consumer.name=messageConsumer
```

Comment:
Is this right? This is configuration for ER right, and that property name should be the same?

Reply:
Ah that was a mistake.